apps/mail/components/mail/mail-list.tsx

```
@@ -20,14 +20,16 @@ import { useParams, useRouter } from 'next/navigation';
 import { useMailNavigation } from '@/hooks/use-mail-navigation';
-import { preloadThread, useThread, useThreads } from '@/hooks/use-threads';
+import { preloadThread, useThreads, useThreads } from '@/hooks/use-threads';
 import { useHotKey, useKeyState } from '@/hooks/use-hot-key';
 import { useSearchValue } from '@/hooks/use-search-value';
 import { markAsRead, markAsUnread } from '@/actions/mail';
 import { ScrollArea } from '@/components/ui/scroll-area';
 import { highlightText } from '@/lib/email-utils.client';
+import { useHotkeysContext } from 'react-hotkeys-hook';
 import { useMail } from '@/components/mail/use-mail';
 import type { VirtuosoHandle } from 'react-virtuoso';
+import { useKeyState } from '@/hooks/use-hot-key';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useKeyState'. Consider to delete it or rename it.

'useKeyState' is defined here:


(lint/suspicious/noRedeclare)